### ISSUE TITLE
Error After React-Native link

### ISSUE THREAD

**Issue by Kodax9** [shoutem/ui#389]
Is there anyone experiencing the same issue after installing @shoutem/ui and react-native link after running the program always error.
* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from
AndroidManifest.xml:11:7-34
    is also present at [me.relex:photodraweeview:1.0.0] AndroidManifest.xml:10:9
-35 value=(true).
  Suggestion: add 'tools:replace="android:allowBackup"' to <application> element
 at AndroidManifest.xml:7:5-24:19 to override.

any body experiencing the same with me ?

**Comment by BogdanTaranenko** [comment]
Yes, I came across this.
just set `android:allowBackup` in _/android/app/src/main/AndroidManifest.xml_ to `true`.
In my case it worked

**Comment by transybao93** [comment]
Change `android:allowBackup` from `false` to `true` worked for me

**Comment by jhaytaslim** [comment]
Yeah,Changing android:allowBackup from false to true worked for me too
